create-loop/package-lock.json

```diff
@@ -1,13 +1,1031 @@
 {
   "name": "@oliveai/create-loop",
-  "version": "1.1.2",
+  "version": "1.1.3",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
     "@oliveai/loop-templates": {
-      "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/@oliveai/loop-templates/-/loop-templates-1.0.0.tgz",
-      "integrity": "sha512-MwfU3oKQMi11C4yPUr++byA6bOajS/iUxfZtWygrcxd8m4cvr9y3Rx7Q3qAWsLQ6Ys+D/s1kVfnS7HfmhHck4Q=="
+      "version": "file:../loop-templates",
```

Comment:
Can you undo these changes to the package-lock since this still references the `file:` version :)